pcsx2/gui/MainFrame.cpp

```diff
@@ -665,7 +665,7 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
-	menubar.Enable( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
+	menubar.Check( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
```

Comment:
Well, thinking again I don't think the issue is specific to GSdx since ZeroGS also exhibits the same behavior.

So a quick fix would be to probably change,

```
menubar.Check( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() )
```

**to**

```
menubar.Check( MenuId_Sys_Shutdown, SysHasValidState())
```